.changeset/breezy-boxes-press.md

```diff
@@ -0,0 +1,19 @@
+---
+'xstate': major
+---
+
+Strict mode is no longer available. Instead, wildcard transitions should be used to explicitly handle undesired events:
```

Comment:
Note that this might require some better wording/grammar:
```suggestion
Strict mode is no longer available. You can no longer control if a machine gets stopped or not when receiving unhandled rejection events from children, nor you can control if a machine throws an error when receiving events with unknown types. Instead of the latter, wildcard transitions should be used to explicitly handle undesired events:
```